mealie/db/models/recipe/recipe_timeline.py

```diff
@@ -0,0 +1,45 @@
+import enum
+from datetime import datetime
+
+from sqlalchemy import Column, DateTime, Enum, ForeignKey, String
+from sqlalchemy.orm import relationship
+
+from .._model_base import BaseMixins, SqlAlchemyBase
+from .._model_utils import auto_init
+from .._model_utils.guid import GUID
+
+
+class TimelineEventType(enum.Enum):
+    system = "system"
+    info = "info"
+    comment = "comment"
```

Comment:
I think I'd prefer to implement these in the schemas folder, and leave the database column as a generic string and let Pydantic handle the validation. That's more consistent with everything else we've done up to this point. Might not be the best, but I'd rather them be consistent.

Reply:
Yeah, makes sense! I was back and forth on whether to make it an enum or string